tests/lib/tools/tests.session

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
 show_help() {
-	echo "usage: tests.session exec [-u USER] [-p PID_FILE] [--] <CMD>"
+	echo "usage: tests.session [-u USER] [-p PID_FILE] exec <CMD>"
```

Comment:
we could have prepare and retore following the same logic
tests.session [-u USER | -u USER1,USER2,...]" prepare | restore

In fact, in most the tests it is used like this

Reply:
We need just break after prepare/restore commands are found